### ISSUE TITLE
i18n is broken

### ISSUE THREAD

**Issue by blinchk** [tenpages/us-level#19]
Ciao,

Looks, that something is wrong with i18n.

**Expected Result**
Map legend labels should be available with i18n, map should be clickable and open menu on click.

**Actual Result**
Screenshot
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/49353043/198661749-b69f6a8b-6a61-45d9-a1ec-3fb9cd8ce710.png">
Console
<img width="782" alt="image" src="https://user-images.githubusercontent.com/49353043/198662558-abb56109-f618-410f-b0be-84808cd37c57.png">
Click on the country do not open pop-up menu.

**Specification**
macOS 13.0
Chrome 106.0.5249.119

**Comment by thesiv95** [comment]
cors issue I guess

**Comment by nkyokoo** [comment]
Do you block Javascript or third party js?

**Comment by blinchk** [comment]
@thesiv95 Yeah, you are right. CORS issues can be easily resolved, if load JS resources (in `scripts` tag) locally, instead of CDN.
